server/base/application.ts

```diff
@@ -14,21 +14,32 @@
  * limitations under the License.
  */
 
+import * as bodyParser from 'body-parser';
 import * as express from 'express';
 import * as path from 'path';
 
 export class Application {
   private static app: express.Application = express();
 
   public static async START(): Promise<void> {
-    await import('../example/example.router');
     this.app.use(express.static(path.join(__dirname, '../client')));
+    this.app.use(bodyParser.json({limit: '10mb'}));
```

Comment:
Do we really need this limit?

Reply:
JSON is limited under 100KB as default.
Default size is not enough to handling big-data through JSON.

Reply:
So, is there any use case in this patch?
If you have an use case, then please add it.
Otherwise, IMHO, remove it.